apps/web/src/common/modules/modals/NotificationEmailModal.vue

```diff
@@ -0,0 +1,276 @@
+                    <p-collapsible-toggle v-if="state.isCollapsed"
+                                          v-model="state.isCollapsed"
+                    >
+                        {{ $t('COMMON.NOTIFICATION_MODAL.COLLAPSE_TITLE') }}
+                    </p-collapsible-toggle>
+                    <p v-if="!state.isCollapsed"
+                       class="collapsed-contents"
+                    >
+                        {{ $t('COMMON.NOTIFICATION_MODAL.COLLAPSE_DESC') }}
+                        <p-button class="send-code-button"
+                                  @click.prevent="handleClickSendEmailButton(true)"
+                        >
+                            <span class="emphasis">{{ $t('COMMON.NOTIFICATION_MODAL.SEND_NEW_CODE') }}</span>
+                        </p-button>
+                    </p>
+                </div>
+            </div>
+        </template>
+        <template #confirm-button>
+            {{ $t('COMMON.NOTIFICATION_MODAL.VERIFY_NOW') }}
+        </template>
+    </p-button-modal>
+</template>
+
+<script lang="ts" setup>
+import {
+    computed, getCurrentInstance, reactive, watch,
+} from 'vue';
+import type { TranslateResult } from 'vue-i18n';
+import type { Vue } from 'vue/types/vue';
+
+import {
+    PButton,
+    PButtonModal,
+    PCollapsibleToggle,
+    PFieldGroup,
+    PI,
+    PIconButton,
+    PTextInput,
+} from '@spaceone/design-system';
+
+import { store } from '@/store';
+
+import { emailValidator } from '@/lib/helper/user-validation-helper';
+import { postValidationCode, postValidationEmail } from '@/lib/helper/verify-email-helper';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+import { useProxyValue } from '@/common/composables/proxy-state';
+
+interface Props {
+    domainId: string
+    userId: string
+    verified: boolean
+    email: string
+    isAdministration: boolean
+    visible: boolean
+}
+
+const props = withDefaults(defineProps<Props>(), {
+    domainId: '',
+    userId: '',
+    verified: false,
+    email: '',
+    isAdministration: false,
+    visible: false,
+});
+
+const vm = getCurrentInstance()?.proxy as Vue;
+
+const emit = defineEmits(['visible', 'handle-user-detail']);
+
+const state = reactive({
+    loading: false,
+    isCollapsed: true,
+    isEditMode: props.verified,
+    myId: computed(() => store.state.user.userId),
+    email: computed(() => (props.isAdministration ? props.email : store.state.user.email)),
+    proxyVisible: useProxyValue('visible', props, emit),
+});
+const formState = reactive({
+    newNotificationEmail: '',
+    verificationCode: '',
+});
+const validationState = reactive({
+    isValidationCodeValid: undefined as undefined | boolean,
+    validationCodeInvalidText: '' as TranslateResult | string,
+});
+
+/* Components */
+const handleEditButton = () => {
+    state.isEditMode = true;
+};
+const handleClickCancel = () => {
+    state.proxyVisible = false;
+    handleReset();
+};
+const handleReset = () => {
```

Comment:
suggestion: This function is not used in template, so it's better to call it `resetFormData()` rather than `handle-`